modelcontextprotocol/server.py

```diff
@@ -380,14 +380,18 @@ def update_assets_tool(
         assets (Union[Dict[str, Any], List[Dict[str, Any]]]): Asset(s) to update.
             Can be a single UpdatableAsset or a list of UpdatableAsset objects.
         attribute_name (str): Name of the attribute to update.
-            Only "user_description" and "certificate_status" are supported.
+            Only "user_description", "certificate_status" and "readme" are supported.
         attribute_values (List[str]): List of values to set for the attribute.
             For certificateStatus, only "VERIFIED", "DRAFT", or "DEPRECATED" are allowed.
+            For readme, the value must be a valid Markdown string.
 
     Returns:
         Dict[str, Any]: Dictionary containing:
             - updated_count: Number of assets successfully updated
             - errors: List of any errors encountered
+    Usage:
+        - If you already have the asset related information, that needs to be passed to the tool, then you can pass the asset as a dictionary. Otherwise call the search_assets tool to get the asset information.
```

Comment:
Any specific reason for adding these usage docs? Was it not working earlier?

Reply:
Resolved via : [bb74f5f](https://github.com/atlanhq/agent-toolkit/pull/63/commits/bb74f5f23aaaa08760b00db8b17484f76bc10a91)